### DISCUSSION TITLE
On Screen Keyboard from Windows

### DISCUSSION THREAD

**Discussion by venom-marvel** [SafeExamBrowser/seb-win-refactoring#175] [Q&A]
Hello everyone. Which setting do I have to set exactly so that I see an on-screen keyboard of Windows 10 as soon as I press in a field to enter.....

Yesterday it worked out once but today I can't do it.

**Comment by dbuechel** [comment]
You'll need to enable the tablet mode in Windows 10 before starting SEB.

**Reply by venom-marvel** [reply]
Hi i have enabled. Tablet mode. But keyboard is not swiping up

**Reply by venom-marvel** [reply]
Do i need to setup something inside seb?

**Reply by dbuechel** [reply]
Which device are you using and how exactly are you trying to activate the keyboard? The activation mechanism unfortunately never worked perfectly, neither in the legacy version 2.x nor in the current version 3.x.

**Reply by venom-marvel** [reply]
I am using Windows 10 pro with the latest seb version. The display on which it is executed is a touch screen. As soon as seb is closed and I click in any field, the keyboard comes out.

Wasn't mentioned that it will run better in version 3 than in 2?

This is another kill feature that I miss. Currently you can only build display kiosk but not an interactive one with virtual keyboard.

**Reply by venom-marvel** [reply]
If it works so badly, why don't you build your own keyboard into it? Shit on window's own keyboard and install yours inside seb.

**Reply by danschlet** [reply]
@dbuechel Maybe he needs to change the Registry setting to use the older, compatible on-screen keyboard? I didn't find it in our documentation (which is currently being updated), probably it was mentioned in an older issue or forum posting.

@venom-marvel You know you also need to activate the "Touch optimized" mode of SEB?

Do you have an idea how much work it is to implement our own on-screen keyboard? We would also need to support multiple languages of course, not only English.

SEB is a non-commercial software, you are free to contribute to its development if you miss some feature...

**Reply by danschlet** [reply]
And btw., we get only very few requests for the tablet mode, as Microsoft and Windows developers failed to make Windows tablets and optimized software really popular. Most people who need a real tablet device use the best ones which exist on the market, iPads. SEB for iOS of course doesn't have such issues...

**Reply by dbuechel** [reply]
@danschlet You're right, in order to work correctly @venom-marvel would also need to set the following key to value `1` in the registry: `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\TabletTip\1.7\DisableNewKeyboardExperience`
